frontend/src/main/scala/bloop/engine/tasks/TestTask.scala

```diff
@@ -43,12 +43,14 @@ object TestTask {
       rawTestOptions: List[String],
       testFilter: String => Boolean,
       handler: LoggingEventHandler,
-      failIfNoTestFrameworks: Boolean,
       mode: RunMode
   ): Task[Int] = {
     import state.logger
+    val isTestProject = project.tags.contains(Tag.Test) || project.tags.contains(
```

Comment:
nit: this reads cleaner IMO
```suggestion
    val isTestProject = project.tags.contains(Tag.Test) ||
      project.tags.contains(Tag.IntegrationTest)

Reply:
manual line breaks around infix operators are needed by scalafmt